### PR TITLE
Test some visual effects on MathML content

### PR DESCRIPTION
**Pull request by rwlbuis**
See https://github.com/mathml-refresh/mathml/issues/179.

### CODE REVIEW

**Comment by fred-wang** [comment]
LGTM, I wonder if some of these things might cause flaky failure due to antialiasing and similar... Maybe use translation instead of or rotation with angle multiple of 90° ;  and a path with horizontal/vertical edges instead of a circle? Not sure if that's needed, and I don't really have any suggestion for blur...

**Review by fred-wang — APPROVED** [review]
see commment above

**Comment by rwlbuis** [comment]
> LGTM, I wonder if some of these things might cause flaky failure due to antialiasing and similar... Maybe use translation instead of or rotation with angle multiple of 90° ; and a path with horizontal/vertical edges instead of a circle? Not sure if that's needed, and I don't really have any suggestion for blur...

Good point about antialiasing, I'll try to prevent that before landing.

**Pull request merged**
